pkg/android/find-android-manifest.go

```diff
@@ -0,0 +1,24 @@
+package android
+
+import "github.com/bugsnag/bugsnag-cli/pkg/utils"
+
+// FindAndroidManifest locates and returns the first existing AndroidManifest.xml path.
+//
+// This function iterates through a list of candidate file paths and returns the
+// first one that exists on disk. This is useful when Android build outputs may
+// place the manifest in multiple variant-specific directories.
+//
+// Parameters:
+//   - paths: A slice of possible manifest file locations.
+//
+// Returns:
+//   - string: The first existing manifest path, or an empty string if none are found.
+func FindAndroidManifest(paths []string) string {
```

Comment:
Could this function rather take the common merged manifest directory and variant name and look in the two known locations?

```
<merged_manifest_path>/<variant>/AndroidManifest.xml
<merged_manifest_path>/<variant>/process<variant>Manfiest/AndroidManifest.xml
```